pytest-wdl-tests/tests/imports/A/main.wdl

```diff
@@ -0,0 +1,16 @@
+version 1.0
+
+# check that we can specify import directories on the command line.
```

Comment:
I guess I would prefer something like "check that we can import relative paths" - since "directories on the command line" is a bit of an implementation detail here?